src/main/resources/stories/test.story

```diff
@@ -0,0 +1,16 @@
+Meta:
```

Comment:
please move the stories package to src/test/resources

Reply:
Moved